File: qt/gen_qtestsupport_widgets.cpp
// +build ignore

#include <QPoint>
#define WORKAROUND_INNER_CLASS_DEFINITION_QTest__QTouchEventWidgetSequence
#include <QWidget>
#include <qtestsupport_widgets.h>
#include "gen_qtestsupport_widgets.h"
QTest__QTouchEventWidgetSequence* QTest__QTouchEventWidgetSequence_new(const QTouchEventWidgetSequence* param1) {
return new QTest::QTouchEventWidgetSequence(*param1);
}
QTouchEventWidgetSequence* QTest__QTouchEventWidgetSequence_Press(QTest__QTouchEventWidgetSequence* self, int touchId, QPoint* pt) {
	return &self->press(static_cast<int>(touchId), *pt);
}
QTouchEventWidgetSequence* QTest__QTouchEventWidgetSequence_Move(QTest__QTouchEventWidgetSequence* self, int touchId, QPoint* pt) {
	return &self->move(static_cast<int>(touchId), *pt);
}
QTouchEventWidgetSequence* QTest__QTouchEventWidgetSequence_Release(QTest__QTouchEventWidgetSequence* self, int touchId, QPoint* pt) {
	return &self->release(static_cast<int>(touchId), *pt);
}
QTouchEventWidgetSequence* QTest__QTouchEventWidgetSequence_Stationary(QTest__QTouchEventWidgetSequence* self, int touchId) {
	return &self->stationary(static_cast<int>(touchId));
}
bool QTest__QTouchEventWidgetSequence_Commit(QTest__QTouchEventWidgetSequence* self, bool processEvents) {
	return self->commit(processEvents);
}
QTouchEventWidgetSequence* QTest__QTouchEventWidgetSequence_Press3(QTest__QTouchEventWidgetSequence* self, int touchId, QPoint* pt, QWidget* widget) {
	return &self->press(static_cast<int>(touchId), *pt, widget);
}
QTouchEventWidgetSequence* QTest__QTouchEventWidgetSequence_Move3(QTest__QTouchEventWidgetSequence* self, int touchId, QPoint* pt, QWidget* widget) {
	return &self->move(static_cast<int>(touchId), *pt, widget);
}
QTouchEventWidgetSequence* QTest__QTouchEventWidgetSequence_Release3(QTest__QTouchEventWidgetSequence* self, int touchId, QPoint* pt, QWidget* widget) {
	return &self->release(static_cast<int>(touchId), *pt, widget);
}
void QTest__QTouchEventWidgetSequence_Delete(QTest__QTouchEventWidgetSequence* self, bool isSubclass) {
	if (isSubclass) {
		delete dynamic_cast<QTest::QTouchEventWidgetSequence*>( self );
	} else {
		delete self;
	}
}

File: qt/gen_qtestsupport_widgets.go
package qt

import (
	"unsafe"
)

type QTest__QTouchEventWidgetSequence struct {
	h          uintptr
	isSubclass bool
}

// NewQTest__QTouchEventWidgetSequence constructs a new QTest::QTouchEventWidgetSequence object.
func NewQTest__QTouchEventWidgetSequence(param1 *QTouchEventWidgetSequence) *QTest__QTouchEventWidgetSequence {
	g := newQTest__QTouchEventWidgetSequence(QTest__QTouchEventWidgetSequence_new(param1))
	g.isSubclass = true
	return g
}

func (this *QTest__QTouchEventWidgetSequence) Press(touchId int, pt *QPoint) *QTouchEventWidgetSequence {
	xxxxxxxxx
}

func (this *QTest__QTouchEventWidgetSequence) Move(touchId int, pt *QPoint) *QTouchEventWidgetSequence {
	xxxxxxxxx
}

func (this *QTest__QTouchEventWidgetSequence) Release(touchId int, pt *QPoint) *QTouchEventWidgetSequence {
	xxxxxxxxx
}

func (this *QTest__QTouchEventWidgetSequence) Stationary(touchId int) *QTouchEventWidgetSequence {
	xxxxxxxxx
}

func (this *QTest__QTouchEventWidgetSequence) Commit(processEvents bool) bool {
	return (bool)(QTest__QTouchEventWidgetSequence_Commit(this.h, (bool)(processEvents)))
}

func (this *QTest__QTouchEventWidgetSequence) Press3(touchId int, pt *QPoint, widget *QWidget) *QTouchEventWidgetSequence {
	xxxxxxxxx
}

func (this *QTest__QTouchEventWidgetSequence) Move3(touchId int, pt *QPoint, widget *QWidget) *QTouchEventWidgetSequence {
	xxxxxxxxx
}

func (this *QTest__QTouchEventWidgetSequence) Release3(touchId int, pt *QPoint, widget *QWidget) *QTouchEventWidgetSequence {
	xxxxxxxxx
}
